### ISSUE TITLE
Bug Report - Default profile not handled

### ISSUE THREAD

**Issue by elken** [microsoft/terminal#2167]
# Environment

```none
Windows build number: [run "ver" at a command prompt]
Microsoft Windows [Version 10.0.18362.175]
Windows Terminal version (if applicable):
0.2.1831.0

Any other software?
```

# Steps to reproduce

1. Change "defaultProfile" to anything other than a valid profile

# Expected behavior

Defaults to an existing profile or uses a default

# Actual behavior

Nothing

**Comment by zadjii-msft** [comment]
Hey wadda ya know, this is actually something that I'm spec'ing up as part of #754 (see spec in #1258).

Thanks for the report though! I'll leave this open to track making sure this gets done.

**Comment by elken** [comment]
Hit it by accident fiddling around with themes and such.

**Comment by zadjii-msft** [comment]
I believe this was actually one of the many things that was resolved by #2422. This was one of the warning cases I decided to include in that initial PR.
